lib/init.coffee

```diff
@@ -1,14 +1,49 @@
+{BufferedProcess, CompositeDisposable} = require 'atom'
+
 module.exports =
   config:
-    elixircExecutablePath:
-      type: 'string'
-      default: ''
-    includeDirs:
-      type: 'string'
-      default: ''
-    pa:
+    executablePath:
       type: 'string'
-      default: ''
+      title: 'Elixirc path'
+      default: '/usr/local/bin/elixirc'
```

Comment:
You might want to default this to just `elixirc` and let the user's environment figure out the real location on their system.